pkg/ddl/backfilling_scheduler.go

```diff
@@ -163,6 +163,7 @@ func initSessCtx(
 	if err := setSessCtxLocation(sessCtx, tzLocation); err != nil {
 		return errors.Trace(err)
 	}
+	sessCtx.GetSessionVars().StmtCtx.InReorg = true
```

Comment:
We need to restore this value in `restoreSessCtx` before putting back to session pool.

Reply:
done